src/makeRobot.js

```diff
@@ -37,8 +37,59 @@
  *
  * @return {object}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'chip version': version,
+    'wheels': wheels,
```

Comment:
```suggestion
    name,
    version,
    wheels,
```
Можна використати ключі з тими ж назвами і зробити скорочений запис.